### PR TITLE
feat(install): Add operator YAML for 1.5.0 in AWS

### PR DESCRIPTION
**Pull request by ranjithwingrider**
- Add openebs operator YAML for making availability of  1.5 in AWS market place

Signed-off-by: Ranjith R <ranjith.raveendran@mayadata.io>

### CODE REVIEW

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 1]
```diff
@@ -0,0 +1,690 @@
...
+      labels:
+        name: maya-apiserver
+        openebs.io/component-name: maya-apiserver
+        openebs.io/version: 1.5.0
+    spec:
+      serviceAccountName: openebs-maya-operator
+      containers:
+      - name: maya-apiserver
+        imagePullPolicy: IfNotPresent
+        image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/m-apiserver:1.5.0
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
        image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/m-apiserver:1.5.0-latest
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 2]
```diff
@@ -0,0 +1,690 @@
...
+        #  value: "/var/openebs"
+        # OPENEBS_IO_LOCALPV_HOSTPATH_DIR can be used to specify the hostpath
+        # to be used for default openebs-hostpath storageclass loaded by OpenEBS
+        # The default path used is /var/openebs/local
+        # This value takes effect only if OPENEBS_IO_CREATE_DEFAULT_STORAGE_CONFIG
+        # is set to true
+        #- name: OPENEBS_IO_LOCALPV_HOSTPATH_DIR
+        #  value: "/var/openebs/local"
+        - name: OPENEBS_IO_JIVA_CONTROLLER_IMAGE
+          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/jiva:1.5.0"
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/jiva:1.5.0-latest"
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 3]
```diff
@@ -0,0 +1,690 @@
...
+        - name: OPENEBS_IO_JIVA_REPLICA_IMAGE
+          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/jiva:1.5.0"
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/jiva:1.5.0-latest"
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 4]
```diff
@@ -0,0 +1,690 @@
...
+        - name: OPENEBS_IO_JIVA_REPLICA_COUNT
+          value: "3"
+        - name: OPENEBS_IO_CSTOR_TARGET_IMAGE
+          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/cstor-istgt:1.5.0"
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/cstor-istgt:1.5.0-latest"
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 5]
```diff
@@ -0,0 +1,690 @@
...
+        - name: OPENEBS_IO_CSTOR_POOL_IMAGE
+          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/cstor-pool:1.5.0"
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/cstor-pool:1.5.0-latest"
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 6]
```diff
@@ -0,0 +1,690 @@
...
+        - name: OPENEBS_IO_CSTOR_POOL_MGMT_IMAGE
+          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/cstor-pool-mgmt:1.5.0"
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/cstor-pool-mgmt:1.5.0-latest"
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 7]
```diff
@@ -0,0 +1,690 @@
...
+        - name: OPENEBS_IO_CSTOR_VOLUME_MGMT_IMAGE
+          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/cstor-volume-mgmt:1.5.0"
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/cstor-volume-mgmt:1.5.0-latest"
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 8]
```diff
@@ -0,0 +1,690 @@
...
+        - name: OPENEBS_IO_VOLUME_MONITOR_IMAGE
+          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/m-exporter:1.5.0"
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/m-exporter:1.5.0-latest"
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 9]
```diff
@@ -0,0 +1,690 @@
...
+        - name: OPENEBS_IO_CSTOR_POOL_EXPORTER_IMAGE
+          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/m-exporter:1.5.0"
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/m-exporter:1.5.0-latest"
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 10]
```diff
@@ -0,0 +1,690 @@
...
+        - name: OPENEBS_IO_HELPER_IMAGE
+          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/linux-utils:1.5.0"
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/linux-utils:1.5.0-latest"
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 11]
```diff
@@ -0,0 +1,690 @@
...
+      labels:
+        name: openebs-provisioner
+        openebs.io/component-name: openebs-provisioner
+        openebs.io/version: 1.5.0
+    spec:
+      serviceAccountName: openebs-maya-operator
+      containers:
+      - name: openebs-provisioner
+        imagePullPolicy: IfNotPresent
+        image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/openebs-k8s-provisioner:1.5.0
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
        image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/openebs-k8s-provisioner:1.5.0-latest
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 12]
```diff
@@ -0,0 +1,690 @@
...
+    metadata:
+      labels:
+        name: openebs-snapshot-operator
+        openebs.io/component-name: openebs-snapshot-operator
+        openebs.io/version: 1.5.0
+    spec:
+      serviceAccountName: openebs-maya-operator
+      containers:
+        - name: snapshot-controller
+          image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/snapshot-controller:1.5.0
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
          image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/snapshot-controller:1.5.0-latest
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 13]
```diff
@@ -0,0 +1,690 @@
...
+            initialDelaySeconds: 30
+            periodSeconds: 60
+        # OPENEBS_MAYA_SERVICE_NAME provides the maya-apiserver K8s service name,
+        # that snapshot controller should forward the snapshot create/delete requests.
+        # If not present, "maya-apiserver-service" will be used for lookup.
+        # This is supported for openebs provisioner version 0.5.3-RC1 onwards
+        #- name: OPENEBS_MAYA_SERVICE_NAME
+        #  value: "maya-apiserver-apiservice"
+        - name: snapshot-provisioner
+          image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/snapshot-provisioner:1.5.0
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
          image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/snapshot-provisioner:1.5.0-latest
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 14]
```diff
@@ -0,0 +1,690 @@
...
+                  fieldPath: metadata.name
+            # the service account of the ndm-operator pod
+            - name: SERVICE_ACCOUNT
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.serviceAccountName
+            - name: OPERATOR_NAME
+              value: "node-disk-operator"
+            - name: CLEANUP_JOB_IMAGE
+              value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/linux-utils:1.5.0"
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
              value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/linux-utils:1.5.0-latest"
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 15]
```diff
@@ -0,0 +1,690 @@
...
+    metadata:
+      labels:
+        app: admission-webhook
+        openebs.io/component-name: admission-webhook
+        openebs.io/version: 1.5.0
+    spec:
+      serviceAccountName: openebs-maya-operator
+      containers:
+        - name: admission-webhook
+          image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/admission-server:1.5.0
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
          image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/admission-server:1.5.0-latest
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 16]
```diff
@@ -0,0 +1,690 @@
...
+      labels:
+        name: openebs-localpv-provisioner
+        openebs.io/component-name: openebs-localpv-provisioner
+        openebs.io/version: 1.5.0
+    spec:
+      serviceAccountName: openebs-maya-operator
+      containers:
+      - name: openebs-provisioner-hostpath
+        imagePullPolicy: Always
+        image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/provisioner-localpv:1.5.0
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
        image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/provisioner-localpv:1.5.0-latest
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 17]
```diff
@@ -0,0 +1,690 @@
...
+        - name: OPENEBS_SERVICE_ACCOUNT
+          valueFrom:
+            fieldRef:
+              fieldPath: spec.serviceAccountName
+        - name: OPENEBS_IO_ENABLE_ANALYTICS
+          value: "true"
+        - name: OPENEBS_IO_INSTALLER_TYPE
+          value: "openebs-aws-operator"
+        - name: OPENEBS_IO_HELPER_IMAGE
+          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/linux-utils:1.5.0"
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
          value: "709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/linux-utils:1.5.0-latest"
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 18]
```diff
@@ -0,0 +1,690 @@
...
+    metadata:
+      labels:
+        name: openebs-ndm-operator
+        openebs.io/component-name: ndm-operator
+        openebs.io/version: 1.5.0
+    spec:
+      serviceAccountName: openebs-maya-operator
+      containers:
+        - name: node-disk-operator
+          image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/node-disk-operator-amd64:v0.4.5
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
          image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/node-disk-operator-amd64:1.5.0-latest
```

File: docs/openebs-operator-1.5.0-aws.yaml [hunk 19]
```diff
@@ -0,0 +1,690 @@
...
+      #
+      # e.g. label the storage nodes with - "openebs.io/nodegroup"="storage-node"
+      # kubectl label node <node-name> "openebs.io/nodegroup"="storage-node"
+      #nodeSelector:
+      #  "openebs.io/nodegroup": "storage-node"
+      serviceAccountName: openebs-maya-operator
+      hostNetwork: true
+      containers:
+      - name: node-disk-manager
+        image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/node-disk-manager-amd64:v0.4.5
```
Comment (muratkars) [inline docs/openebs-operator-1.5.0-aws.yaml]:
```suggestion
        image: 709373726912.dkr.ecr.us-east-1.amazonaws.com/3ed4f202-6b8f-4bc9-adaa-20ce0807bba3/cg-1202108174/quay.io/openebs/node-disk-manager-amd64:1.5.0-latest
```

File: docs/openebs-operator-1.5.0-aws.yaml (hunk 1 above)
Comment (ranjithwingrider) [reply docs/openebs-operator-1.5.0-aws.yaml]: Addressed in single commit

File: docs/openebs-operator-1.5.0-aws.yaml (hunk 2 above)
Comment (ranjithwingrider) [reply docs/openebs-operator-1.5.0-aws.yaml]: Addressed in single commit

**Comment by ranjithwingrider** [comment]
@muratkars I have addressed the requested changes in single PR. Please review now..

**Pull request merged**